src/ui/frmpreferences.ui

```diff
@@ -463,8 +464,8 @@
              <rect>
               <x>0</x>
               <y>0</y>
-              <width>403</width>
-              <height>196</height>
+              <width>395</width>
+              <height>205</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
```

Comment:
There's a new Widget in line 439. The page widget for the shortcuts page, called page_5.

But the other changed tags? Yea, they sometimes sneak in :x. I don't really know what prompted the designer to subtly change the height..